app/services/pages/column.rb

```diff
@@ -0,0 +1,18 @@
+module Pages
+  class Column
+
+    attr_reader :widgets
+
+    def initialize
+      @widgets = []
+    end
+
+    def find_widget(widget_name)
+      @widgets.select { |w| w.name == widget_name.to_sym }.first
+    end
+
+    def <<(widget)
+      @widgets << widget
+    end
+  end
+end
```

Comment:
Now I see that widgets are being put into Column instances. Does this mean only table-oriented layouts are supported?

I'd expect either free-form layout where each snippet adds its own `col-x-y`